Tests/SwiftDocCTests/Diagnostics/DiagnosticConsoleWriterTests.swift

```diff
@@ -127,28 +127,329 @@ class DiagnosticConsoleWriterTests: XCTestCase {
+            let consumer = DiagnosticConsoleWriter(logger, highlight: true)
+            let diagnostic = Diagnostic(source: source, severity: .warning, range: range, identifier: identifier, summary: summary, explanation: explanation)
+            let problem = Problem(diagnostic: diagnostic, possibleSolutions: [])
+            consumer.receive([problem])
+            try? consumer.finalize()
+            XCTAssertEqual(logger.output, """
+            \u{001B}[1;33mwarning: \(summary)\u{001B}[0;39m
+            \(explanation)
+            \(expectedPath)
+            """)
+        }
+
+        do {
+            let logger = Logger()
+            let consumer = DiagnosticConsoleWriter(logger, highlight: true)
+            let diagnostic = Diagnostic(source: source, severity: .hint, range: range, identifier: identifier, summary: summary, explanation: explanation)
+            let problem = Problem(diagnostic: diagnostic, possibleSolutions: [])
+            consumer.receive([problem])
+            try? consumer.finalize()
+            XCTAssertEqual(logger.output, """
+            \u{001B}[1;39mnotice: \(summary)\u{001B}[0;39m
+            \(explanation)
+            \(expectedPath)
+            """)
+        }
+
+        do {
+            let logger = Logger()
+            let consumer = DiagnosticConsoleWriter(logger, highlight: true)
+            let diagnostic = Diagnostic(source: source, severity: .information, range: range, identifier: identifier, summary: summary, explanation: explanation)
+            let problem = Problem(diagnostic: diagnostic, possibleSolutions: [])
+            consumer.receive([problem])
+            try? consumer.finalize()
+            XCTAssertEqual(logger.output, """
+            \u{001B}[1;39mnote: \(summary)\u{001B}[0;39m
+            \(explanation)
+            \(expectedPath)
+            """)
+        }
+    }
+
+    func testDefaultFormatting_DisplaysRelativePath() {
+        let baseURL = URL(string: "/path/to")!
+        let source = URL(string: "/path/to/file.md")!
+        let range = SourceLocation(line: 1, column: 8, source: source)..<SourceLocation(line: 10, column: 21, source: source)
+        let identifier = "org.swift.docc.test-identifier"
+        let summary = "Test diagnostic summary"
+        let explanation = "Test diagnostic explanation."
+
+        let logger = Logger()
+        let consumer = DiagnosticConsoleWriter(logger, baseURL: baseURL, highlight: true)
+        let diagnostic = Diagnostic(source: source, severity: .warning, range: range, identifier: identifier, summary: summary, explanation: explanation)
+        let problem = Problem(diagnostic: diagnostic, possibleSolutions: [])
+        consumer.receive([problem])
+        try? consumer.finalize()
+        XCTAssertEqual(logger.output, """
+        \u{001B}[1;33mwarning: \(summary)\u{001B}[0;39m
+        \(explanation)
+        --> file.md:1:8-10:21
+        """)
+    }
+
+    func testDefaultFormatting_DisplaysNotes() {
```

Comment:
nit: There are enough of these default formatting tests that it probably makes sense to move them all to a separate file so that they wouldn't need the "DefaultFormatting_" prefix.

Reply:
Right, fixed [here](https://github.com/apple/swift-docc/pull/535/commits/2536b61af4daf7e31ffffa195de0bf5e4c3d25e0)!